walkthroughs/howto-cross-account/secondary-account/deploy.sh

```diff
@@ -86,6 +86,8 @@ delete_cfn_stack() {
 }
 
 delete_stacks() {
+    echo "Deleting route to cleanly delete ${PROJECT_NAME}-mesh"
+    aws --profile ${AWS_PRIMARY_PROFILE} appmesh delete-route --mesh-name ${PROJECT_NAME}-mesh --virtual-router-name backend-vr --route-name backend-route
```

Comment:
Why is this required?

Reply:
On deleting the mesh stack, I was getting `ResourceInUse` exception. This is because we are updating the route (resource of primary account) to hit the application deployed in the secondary account. So I am deleting the route in the mesh of the primary account before deleting the mesh in the secondary account.